recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "cdms2" %}
-{% set version = "2.12" %}
+{% set version = "3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/UV-CDAT/cdms/archive/v{{ version }}.tar.gz
-  sha256: 0aba8451521010d05c1317d21ad0e76a2fca8a5533b05a0f88af920a47ae2456
+  url: https://github.com/UV-CDAT/cdms/archive/master.tar.gz
```

Comment:
This will cause the package to be built from the master, it might be more stable to build from a tagged version.